ndn-svs/svspubsub.hpp

```diff
@@ -198,16 +199,31 @@ class SVSPubSub : noncopyable
   void
   onRecvExtraData(const Block& block);
 
+  void
+  insertMapping(const NodeID& nid, const SeqNo seqNo, const Name& name);
+
+  void
+  fetchAll();
+
+  void
+  cleanUpFetch(const std::pair<Name, SeqNo>& publication);
+
 public:
   static inline const Name EMPTY_NAME;
+  static inline const size_t MAX_DATA_SIZE = 8000;
+  static inline const time::milliseconds FRESH_FOREVER = time::years(10000); // well ...
 
 private:
+  Face& m_face;
   const Name m_syncPrefix;
   const Name m_dataPrefix;
   const UpdateCallback m_onUpdate;
   const SecurityOptions m_securityOptions;
   SVSync m_svsync;
 
+  // Null validator for segment fetcher
+  ndn::security::v2::ValidatorNull m_nullValidator;
```

Comment:
Please don't use the `v2` namespace, it's an inline namespace and effectively an implementation detail.
The public API is `ndn::security::ValidatorNull`

Reply:
Right.